Methods/Methods/main.m

```diff
@@ -13,13 +13,20 @@ int main(int argc, const char * argv[]) {
         
         NSString *myString = @"I feel like we've greeted the world enough already so we'll just stick with a casual nod at this point and move past the formal greetings.";
         
-        NSString *shoutyString = @""; // Set shoutyString to the upper case version of myString.
+        NSString *shoutyString = [myString uppercaseString]; // Set shoutyString to the upper case version of myString.
+        NSLog(@"Print assigned value of shoutyString: %@", shoutyString);
+        //code
+        //shoutyString.
         
-        NSInteger stringLength = 0; // Set stringLength to the length of myString
+        NSInteger stringlength = [myString length]; // Set stringLength to the length of myString
+        NSLog(@"Print value of stringlength: %ld", (long)stringlength); // XCODE autocorrected this line and line 28, could you explain why
```

Comment:
"OS X uses several data types—NSInteger, NSUInteger,CGFloat, and CFIndex—to provide a consistent means of representing values in 32- and 64-bit environments. In a 32-bit environment, NSInteger and NSUInteger are defined as int and unsigned int, respectively. In 64-bit environments, NSInteger and NSUInteger are defined as long and unsigned long, respectively. To avoid the need to use different printf-style type specifiers depending on the platform, you can use the specifiers shown in Table 3. Note that in some cases you may have to cast the value."
Taken from the 'Platform Dependencies' section of [Apple's String Format Specifiers](https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/Strings/Articles/formatSpecifiers.html)
Basically, %d is for 32-bit integers which only applies to NSInteger in 32-bit environments (most modern systems are 64-bit).